### ISSUE TITLE
react-imported-component: unknown mark

### ISSUE THREAD

**Issue by garthenweb** [theKashey/react-imported-component#173]
Since we migrated from v5 to v6 we get errors regarding the hydration marks, e.g. 

```
Uncaught Error: react-imported-component: unknown mark(-1saegod) has been used. Client and Server should have the same babel configuration.
 ```

This only happens for nested dynamic imports. So in case we have a dynamic import, which contains another dynamic import, the error is thrown during
```js
rehydrateMarks(window.___REACT_DEFERRED_COMPONENT_MARKS)
```

Dynamic imports for the first level work fine.

I tried to update the babel config (as we had minor differences), but even with a super simple config we get the issue.

``` js
module.exports = {
  plugins: [
    'react-imported-component/babel',
    'react-hot-loader/babel',
  ],
  presets: [
    '@babel/preset-react',
    [
      '@babel/preset-env',
      {
        useBuiltIns: 'usage',
        corejs: '2',
      },
    ],
  ],
}
```

We use Parcel to bundle our code. We have two different entry points, one for the server and one for the client. In general, the application code within the dynamic bundles is the same.

Any idea how this can be solved?

**Comment by theKashey** [comment]
`v5` did not reported issues like this, what's why you haven't seen the error before.

First of all - please try `@next` (6.2.4) version - it might have everything already fixed. If not - then I would ask you to give me your `async-requires`  (the file generated by `imported-componet`) to check.

**Comment by garthenweb** [comment]
@theKashey Thanks for the hint. The `@next` version did not make any difference.

I tried to reproduce this based on a simple example. I found out, the errors happen when `async-requires` are never imported within the client bundle. I was not aware that this is a required step.

I didn't try adding it yet but I am comfortable that it will solve the issue. Therefore closing this issue (for now).

PS: While debugging I reworked the example project as it was not working for me. See https://github.com/theKashey/react-imported-component/pull/174
